content/en/blog/2023/native-sidecars/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Kubernetes Native Sidecars in Istio
+description: Demoing the new SidecarContainers feature with Istio.
+publishdate: 2023-07-25
+attribution: "John Howard (Google)"
+keywords: [istio,sidecars,kubernetes]
+---
+
+If you have heard anything about service meshes, it is that they work using the sidecar pattern: a proxy server is deployed alongside your application code.
+The sidecar pattern is just that: a pattern.
+Up until this point, there has been no formal support for 'sidecar containers' in Kubernetes at all.
+
+This has caused no end of problems: what if you have a job that terminates by design, but a sidecar container that doesn't?
+This exact use case is the [most popular ever on the Kubernetes issue tracker](https://github.com/kubernetes/kubernetes/issues/25908).
+
+A formal proposal for adding sidecar support in Kubernetes was raised in 2019, with many stops and starts along the way.
+After a reboot of the project last year, formal support for sidecars is being released to Alpha in Kubernetes 1.28.
+Istio has implemented support for this feature, and in this post you can learn how to take advantage of it.
+
+## Sidecar woes
+
+Sidecar containers give a lot of power, but come with some issues.
+While containers within a Pod can share some things, their *lifecycle's* are entirely decoupled.
+To Kubernetes, both of these containers are functionally the same.
+
+However, in Istio they are not the same - the Istio container is required for the primary application container to run,
+and has no value without the primary application container.
+
+This mismatch in expectation leads to a variety of issues:
+* If the application container starts faster than Istio's container, it cannot access the network.
+  This wins the [most +1's](https://github.com/istio/istio/issues/11130) on GitHub by a landslide.
+* If Istio's container shuts down before the application container, it cannot access the network.
+  This often means we shutdown too fast (breaking the network) or too slow (leaving pods running with no purpose).
```

Comment:
I might think about removing this sentence. It's just a recap of the first two items and sort of separates the rest from this list.